### ISSUE TITLE
Create_rom_lists.py for XBOX games

### ISSUE THREAD

**Issue by jtretheway16** [Rocky5/XBMC-Emustation#46]
Hello JCRocky5,

I keep getting an error when trying manual scan my XBOX games.  They are on my F: and G: drives in a "Games" folder.  XBMC is on the E: drive, with the emulators, roms, and media on the F.

I've attached the log, let me know if you can't see it.  You're the man, uh man, but seriously, I love using your creation and have been able to learn so much from you and the rest of the community over the last two years.

[Create_roms_lists.py Error.txt](https://github.com/Rocky5/XBMC-Emustation/files/5289547/Create_roms_lists.py.Error.txt)

Respectfully,
Jeremy

**Comment by Rocky5** [comment]
One of your games xbe files are corrupt. It’s failing at extraction the info from the xbe. 

So all you can do is watch until it errors and check what the last game and the one after xbe file. 

I will be adding to the script to output the game name during the scan, but that will only help if the xbe isn’t corrupt or damaged. (But it will narrow down the user looking as it will be the next game in the list of what’s been entered in the log)

**Comment by jtretheway16** [comment]
Thanks for the reply, I had already erased the games from both drives, so I couldn't narrow it down from another scan.  However, with no games, I put the first one on by copying an actual disc to the hdd (X-Men Legends); when i manually scanned the system, it worked fine.  I then proceeded to add each game individually via FTP, doing a manual scan after each download to ensure the file wasn't corrupt.  

Idk what the deal is, but I was able to add the same 14 games that were originally there, and now there's no problems.  Either way, thanks for your time and response.  Having the output game name in the script will certainly ease a lot of pain, especially when I start getting my console running again (this was for my son's XBOX).  14 games wasn't that painful, but I have close to 100 games to scan in lol.

You're the best!  Take care man.
